rollingpin/frontends.py

```diff
@@ -270,15 +270,18 @@ def on_precheck(self):
 
         bad_time = status["time_status"] not in ("work_time", "cleanup_time")
         deploy_in_progress = status["busy"]
+        hold = status.get("hold")
```

Comment:
🔕 The variable name makes this read like a boolean to me until below where I realize it's actually a string.